### PR TITLE
Document how to build and push the node services docker image

### LINKED ISSUE #241: Document how the e2e-node-services Docker image is being used and updated
Document how the e2e-node-services Docker image is being used and updated.

### PR DESCRIPTION
**Pull request by jackkleeman**
And also how to update the Lambda

Fixes #241

### CODE REVIEW

File: services/node-services/README.md [hunk 1]
```diff
@@ -14,10 +14,19 @@ To build:
...
-$ gradle :services:node-services:dockerBuild
+$ docker build --platform linux/arm64,linux/amd64 -t ghcr.io/restatedev/e2e-node-services --push .
+```
+
+## Update the Lambda function
+A node services Lambda is used by the Lambda verification tests.
+
+```shell
+npm run bundle
+aws lambda update-function-code --function-name 'arn:aws:lambda:eu-central-1:663487780041:function:e2e-node-services' --zip-file fileb://dist/index.zip
```
Comment (tillrohrmann) [inline services/node-services/README.md:29]: Maybe also state when to create a new release. I assume it is required every time a breaking change happens in one of the SDKs.

How will we prevent this from being forgotten (like me doing a change in the SDK and then forgetting about it)?

**Review by tillrohrmann — COMMENTED** [review]
Thanks for creating this PR @jackkleeman. I am wondering how we can prevent us from forgetting to update this image when we do SDK breaking changes?

File: services/node-services/README.md (hunk 1 above)
Comment (jackkleeman) [reply services/node-services/README.md:29]: well, if its a breaking change, you know to do it when the tests start failing, which is generally when i do it. if it was some other subtle sdk change that we wanted to test in the verif cluster, we would need to automate this to truly avoid it. i dont think its really that important right now though

Comment (tillrohrmann) [reply services/node-services/README.md:29]: Fair enough. Merging this PR now.

**Review by tillrohrmann — APPROVED** [review]
LGTM. Merging this PR now.

**Pull request merged**